src/Component/CodeEditor/CodeEditor.tsx

```diff
@@ -163,22 +169,17 @@ export const CodeEditor: React.FC<CodeEditorProps> = ({
 
   const onChange = async(v: string) => {
     setValue(v);
-    setInvalidMessage(undefined);
+    setReadStyleResult(undefined);
     try {
       let parsedStyle;
       if (activeParser) {
-        const { output, errors = [] } = await activeParser.readStyle(v);
-        if (errors.length > 0) {
-          setInvalidMessage(errors.map(e => e.message).join(', '));
-        } else {
-          parsedStyle = output;
-        }
+        setReadStyleResult(await activeParser.readStyle(v));
       } else {
         parsedStyle = JSON.parse(v);
       }
       onStyleChange(parsedStyle);
     } catch (err) {
-      setInvalidMessage(err.message);
+      // TODO:
```

Comment:
Will the implementation of this be part of the current PR?

Reply:
[d94c48d](https://github.com/geostyler/geostyler/pull/1741/commits/d94c48d652110f745f1cf07cbb5835b394c73fb8)